py_ecc/bls12_381/bls12_381_curve.py

```diff
@@ -63,7 +63,7 @@ def is_inf(pt: GeneralPoint[Field]) -> bool:
 def is_on_curve(pt: Point2D[Field], b: Field) -> bool:
     if is_inf(pt):
         return True
-    x, y = pt
+    x, y = pt  # type: ignore[misc]
```

Comment:
If anyone knows why mypy is so unhappy here, please let me know. The `if is_inf(pt)` check above checks for `None` so there is no way to reach line 66 with `pt = None`. MyPy doesn't realise this for some reason. :/